lib/razorpay/payment.rb

```diff
@@ -10,6 +10,46 @@ class Payment < Entity
     def self.request
       Razorpay::Request.new('payments')
     end
+    
+    def self.createRecurringPayment(data={})
+      request.post "create/recurring" , data
+    end
+
+    def self.createJsonPayment(data={})
+      request.post "create/json" , data
+    end
+
+    def self.fetchPaymentDowntime()
+      request.get "downtimes"
+    end
+
+    def self.fetchPaymentDowntimeById(id)
+      request.get "downtimes/#{id}"
+    end
+
+    def self.fetchCardDetails(id)
+      request.get "#{id}/card"  
+    end
+    
+    def fetchTransfer
+      self.class.request.get "#{id}/transfers"  
```

Comment:
Pls check from owners of transfer route about the uri - https://razorpay.com/docs/api/payments/route/#fetch-transfers-for-a-payment
From docs it is mentioned that `status` is mandatory

Reply:
doc is updated now